### PR TITLE
Fix: Return early

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] returns early

### CODE REVIEW

File: src/Composer/Package/Version/VersionGuesser.php [hunk 1]
```diff
@@ -63,26 +63,28 @@ public function __construct(Config $config, ProcessExecutor $process, SemverVers
      */
     public function guessVersion(array $packageConfig, $path)
     {
-        if (function_exists('proc_open')) {
-            $versionData = $this->guessGitVersion($packageConfig, $path);
-            if (null !== $versionData && null !== $versionData['version']) {
-                return $this->postprocess($versionData);
-            }
+        if (!function_exists('proc_open')) {
+            return;
```
Comment (localheinz) [inline src/Composer/Package/Version/VersionGuesser.php]: Should we explicitly return `null` here?

```suggestion
            return null;
```

File: src/Composer/Package/Version/VersionGuesser.php [hunk 2]
```diff
@@ -63,26 +63,28 @@ public function __construct(Config $config, ProcessExecutor $process, SemverVers
...
-            }
+        $versionData = $this->guessFossilVersion($packageConfig, $path);
+        if (null !== $versionData && null !== $versionData['version']) {
+            return $this->postprocess($versionData);
+        }
+
+        $versionData = $this->guessSvnVersion($packageConfig, $path);
+        if (null !== $versionData && null !== $versionData['version']) {
+            return $this->postprocess($versionData);
         }
```
Comment (localheinz) [inline src/Composer/Package/Version/VersionGuesser.php]: Should we explicitly return `null` here?

```suggestion
        }
        
        return null;
```

**Comment by Seldaek** [comment]
Thanks

**Comment by localheinz** [comment]
Thank you, @Seldaek!

**Pull request merged**